lib/sms_notificaiton.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+
+
+class SmsNotification
+def order
+account_sid = "REMOVED"
+auth_token = "REMOVED"
+
+delivery_time = Time.now + 3600
+
+begin
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    @client.account.messages.create({
+        :from => 'REMOVED',
+        :to => 'REMOVED',
+        :body => "Thankyou for ordering through command line takeaways! Your order will be with you by #{delivery_time.hour}:#{'%02d' %delivery_time.min}."
```

Comment:
Metrics/LineLength: Line is too long. [155/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/FormatString: Favor `format` over `String#%`. (https://github.com/bbatsov/ruby-style-guide#sprintf)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)